src/lib/dfp/display-ads.ts

```diff
@@ -21,8 +22,14 @@ const displayAds = (): void => {
 	window.googletag.pubads().collapseEmptyDivs();
 	window.googletag.enableServices();
 
-	loadAdvert(dfpEnv.advertsToLoad[0]);
-	dfpEnv.advertsToLoad.length = 0;
+	const firstAdvertToLoad: Advert | undefined = dfpEnv.advertsToLoad.length
+		? dfpEnv.advertsToLoad[0]
+		: undefined;
```

Comment:
I think shortening this to
```suggestion
	const firstAdvertToLoad = dfpEnv.advertsToLoad[0];
```
Should still work